eox_nelp/middleware.py

```diff
@@ -93,18 +110,19 @@ def __call__(self, request):
 
         return self.get_response(request)
 
-    def _generate_handler(self, field):
+    def _generate_handler(self, field, label=None):
         """Every field will require a handler method, check the logic here
         https://github.com/eduNEXT/edunext-platform/blob/ednx-release/mango.master.nelp/
         openedx/core/djangoapps/user_authn/views/registration_form.py#L360,
         this method generates a specific handler for each new field with its specific label.
 
         Args:
             field: String
+            label: String or None
         Returns:
             func: Handler method
         """
-        label = field.capitalize()
+        label = label.capitalize()
```

Comment:
@andrey-canon do you think that in this way we can avoid this commit ?
https://github.com/eduNEXT/edunext-platform/commit/acc844c166ff381c0b9c46a2dde2b1572d59664a

Reply:
I don't think so